packages/web/src/app/doc-pages/components/components-doc.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="e-title-md e-mb-40">Components</div>
+
+<elvia-radio-filter
+  [value]="filterValue"
+  [name]="'componentFilter'"
+  [items]="[
+    { label: 'All', value: 'all' },
+    { label: 'Navigation', value: 'navigation' },
+    { label: 'Action', value: 'action' },
+    { label: 'Input', value: 'input' },
+    { label: 'Data display', value: 'dataDisplay' },
+    { label: 'Feedback', value: 'feedback' },
+    { label: 'Layout', value: 'layout' }
+  ]"
+  (valueOnChange)="handleCategoryChange($any($event).target.value)"
```

Comment:
Would this work?
```suggestion
  (valueOnChange)="filterValue=($any($event).target.value)"
```
then we could delete the `handleCategoryChange` method